### ISSUE TITLE
Exported classes with static class properties error

### ISSUE THREAD

**Issue by melck** [babel/babel#2868]
**Input:**

```javascript
export class MyClass {
  static property = value;
}

```

**Stack trace:**

```
Error: index.js: We don't know what to do with this node type. We were previously a Statement but we can't fit in here?
    at NodePath.insertAfter (/.../node_modules/babel-traverse/lib/path/modification.js:181:13)
    at PluginPass.Class (/.../node_modules/babel-plugin-transform-class-properties/lib/index.js:146:14)
    at /.../node_modules/babel-traverse/lib/visitors.js:271:19
    at NodePath._call (/.../node_modules/babel-traverse/lib/path/context.js:72:18)
    at NodePath.call (/.../node_modules/babel-traverse/lib/path/context.js:44:17)
    at NodePath.visit (/.../node_modules/babel-traverse/lib/path/context.js:102:12)
    at TraversalContext.visitQueue (/.../node_modules/babel-traverse/lib/context.js:151:16)
    at TraversalContext.visitSingle (/.../node_modules/babel-traverse/lib/context.js:111:19)
    at TraversalContext.visit (/.../node_modules/babel-traverse/lib/context.js:195:19)
    at Function.traverse.node (/.../node_modules/babel-traverse/lib/index.js:139:17)
```

Example repo: https://github.com/babel/babel/issues/2868

**Comment by hzoo** [comment]
This might be the same as https://github.com/babel/babel/issues/2722? At least https://github.com/babel/babel/issues/2722#issuecomment-152850484

**Comment by thejameskyle** [comment]
Updating this issue and choosing it in favor of others because it accurately describes the issue without too much noise. Locking it so people don't fill it with noise.

**Comment by sebmck** [comment]
Fixed via 54c13eef5cd693a58bafa43d59d8fbee2600a192.
